lib/vips/image.rb

```diff
@@ -25,35 +25,35 @@ module Vips
   attach_function :vips_foreign_find_load_buffer, [:pointer, :size_t], :string
   attach_function :vips_foreign_find_save_buffer, [:string], :string
 
-  if Vips::at_least_libvips?(8, 9)
+  if Vips.at_least_libvips?(8, 9)
```

Comment:
I don't really like this rule either --- it's `Vips::Image.new_from_file`, but `Vips.at_least?`. This feels inconsistent.

It ought to always be Module::method, I think. But perhaps we just have to live with adding some ugliness.

Reply:
`Module::method` it's very uncommon. At least on the projects where I have been.

Reply:
Hmmm, but `Module::method` is just using `::` for namespace resolution, isn't it? It's not a method call. It's exactly like `Module::Class`. Or does the `::` refer to the thing to the *right* of the operator? That seems weird.

I think it's ugly and illogical, but if it's common practice, I can see the advantage.